blog/2023/02/the-injectivity-radii-of-the-unitary-groups.md

```diff
@@ -0,0 +1,206 @@
+This maximum distance between the initial and final position is called the _injectivity radius_[^1].
+
+The question motivating this post is, what is the injectivity radius for the rotations in not just 2 and 3 but any dimension?
+And more generally, what is it for the [unitary group](https://en.wikipedia.org/wiki/Unitary_group) and its most common subgroups?
+Since these groups are featured in many (all?) of the introductory texts on Lie groups and manifolds, and since the injectivity radius is a basic property introduced in differential geometry textbooks, I was surprised that I could not find a single reference giving these radii for these groups.
+
+In this post I'll work out these radii.
+Marvelously, we don't need any differential geometry or group theory to do this, just linear algebra!
+Nevertheless, this post assumes familiarity with these topics and for the sake of space will try not define all common terms or notation.[^2]
+
+## The injectivity radius
+
+Consider a point $p$ on some manifold $\mathcal{M}$ with tangent vectors $X,Y \in T_p \mathcal{M}$.
+Assume a Riemannian metric $g_p$ defining an inner product $g_p\colon (X, Y) \mapsto \ip{X}{Y}_g$, which induces a norm $\norm{X}_g$.
+
+Let's denote the exponential map $\exp_p\colon X \mapsto q$ for $q \in \mathcal{M}$ and the logarithmic map $\log_p\colon q \mapsto Y$.
+The injectivity radius at $p$ is defined as the norm of the smallest $X$ for which $X \ne Y = \log_p (\exp_p X)$, or in other words, the smallest $X$ for which the logarithmic map no longer is the inverse of the exponential map.
+We further define the global injectivity radius of $\mathcal{M}$ as the infimum of the injectivity radii at all points on the manifold.
+
+Notationally, we define this global injectivity radius as
+$$\operatorname{inj}^-_{\mathcal{M}} = \inf_{(p, X) \in T \mathcal{M}}\{\norm{X}_g | \log_p(\exp_p(X)) \ne X\}.$$
+
+We'll also consider the related supremum
+$$\operatorname{inj}^+_{\mathcal{M}} = \sup_{(p, X) \in T \mathcal{M}}\{\norm{X}_g | \log_p(\exp_p(X)) = X\}.$$
+These two quantities form the lower and upper bound radii, respectively, of two geodesic balls within which the exponential map is invertible.
+
+## The unitary group(s)
+
+The Unitary group $\mathrm{U}(n, \mathbb{F})$ over some number system $\mathbb{F}$ is the group of all $n \times n$ matrices $p \in \mathbb{F}^{n \times n}$ for which $p^\mathrm{H} p = I_n$, where ${\cdot}^\mathrm{H}$ denotes the matrix adjoint.
+$\mathbb{F}$ could be the real numbers $\R$, complex numbers $\C$, or quaternions $\H$.
+
+We will also deal with the following subgroups:
+- $\mathrm{SU}(n)$: the special unitary group, which consists of complex unitary matrices whose determinant is +1
+- $\mathrm{O}(n) \equiv \mathrm{U}(n, \R)$: the orthogonal group, i.e. the group of rotations and reflections
+- $\mathrm{SO}(n)$: the special orthogonal group, i.e. the group of real rotations
+
```

Comment:
Maybe one could introduce the tangent space a bit more detailed here as well? You use X,Y from there in the next paragraph.